tests/consistent_replicate_local/run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -e
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+SINK_TYPE=$1
+
+# check resolved ts has been persisted in redo log meta
+function check_resolved_ts() {
+	changefeedid=$1
+	check_tso=$2
+	redo_dir=$3
+	rts=$(cdc redo meta --storage=local --dir="$redo_dir" | grep -oE "resolved-ts:[0-9]+" | awk -F: '{print $2}')
+	if [[ "$rts" -gt "$check_tso" ]]; then
+		return
+	fi
+	echo "global resolved ts $rts not forward to $check_tso"
+	exit 1
+}
+
+export -f check_resolved_ts
+
+function run() {
+	# TODO: sink kafka sink, since this is a MySQL eventual consistent test
```

Comment:
Sorry, I don't quite understand the comment?

Reply:
Update the comment, at first I meant in the future, we may support eventually consistent replication with the link TiCDC -> Kafka -> TiDB/MySQL.
However we only need to support eventually consistent with MySQL sink now, so I changed the comment according to the real situation